Libraries/libcar/Sources/Rendition.cpp

```diff
@@ -227,7 +188,9 @@ Load(
                         };
                         slices.push_back(slice);
                 }
-                rendition.slices() = std::move(slices);
+                if (slices.size() > 0) {
```

Comment:
The point is that I should be initializing the slices, and then I can take out the special case in the write function.